### ISSUE TITLE
Typescript 5.0.X support

### ISSUE THREAD

**Issue by oteoe** [kettanaito/msw#1567]
### Scope

Adds a new behavior

### Compatibility

- [ ] This is a breaking change

### Feature description

Typescript 5.0.2 is out

could you please update the dependency for:

`npm ERR!   peerOptional typescript@">= 4.4.x <= 4.9.x" from msw@1.1.0`

**Comment by mattcosta7** [comment]
there's already a PR up for this

**Comment by Maxim-Mazurok** [comment]
Will be using a patch with `npx patch-package` for now:

```patch
TODO: remove this after https://github.com/mswjs/msw/pull/1566 is merged

diff --git a/node_modules/msw/package.json b/node_modules/msw/package.json
index 21feef5..5c93117 100644
--- a/node_modules/msw/package.json
+++ b/node_modules/msw/package.json
@@ -143,7 +143,7 @@
     "webpack-http-server": "^0.5.0"
   },
   "peerDependencies": {
-    "typescript": ">= 4.4.x <= 4.9.x"
+    "typescript": ">= 4.4.x <= 5.0.x"
   },
   "peerDependenciesMeta": {
     "typescript": {
```

**Comment by Maxim-Mazurok** [comment]
Tried to patch it with `npx patch-package`, but npm throws before even installing, so it doesn't work, will wait for https://github.com/mswjs/msw/pull/1566 to get published, thanks!

**Comment by mattcosta7** [comment]
> Tried to patch it with `npx patch-package`, but npm throws before even installing, so it doesn't work, will wait for https://github.com/mswjs/msw/pull/1566 to get published, thanks!

A --force flag should let the install happen, then patch and reinstall (haven't tested, but should work I think)

**Comment by kettanaito** [comment]
## Released: v1.1.1 🎉

This has been released in v1.1.1!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v1.1.1)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/1.1.1)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
